### PR TITLE
DT-6358 implementation for "to trip" button for navigation.    

### PR DESCRIPTION
**Pull request by SamiPuttonen**
## Proposed Changes

  - Button to mobile view which opens new navigator view.  

## Pull Request Check List

  - A reasonable set of unit tests is included
  - Console does not show new warnings/errors
  - Changes are documented or they are self explanatory
  - This pull request does not have any merge conflicts
  - All existing tests pass in CI build

## Review

  - Read and verify the code changes
  - Test the functionality by running the UI locally with all popular browsers available in your platform
  - Check that the implementation matches the design, when such one is defined in a Jira issue
  - Merge the pull request

### CODE REVIEW

File: app/component/itinerary/ItineraryDetails.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import TicketInformation from './TicketInformation';
 import ItinerarySummary from './ItinerarySummary';
 import Legs from './Legs';
 import BackButton from '../BackButton';
-import SecondaryButton from '../SecondaryButton';
+import ToTripButton from './ToTripButton';
```
Comment (vesameskanen) [inline app/component/itinerary/ItineraryDetails.js]: change name to NavigatorButton

File: app/component/itinerary/ItineraryDetails.js [hunk 2]
```diff
@@ -293,16 +293,13 @@ class ItineraryDetails extends React.Component {
...
-            this.props.setNavigation && false && (
-	      // TODO: placeholder for real component, change this
+
+            this.props.setNavigation && (
               <div key="tracking">
-                <SecondaryButton
-                  buttonName="tracking"
-                  ariaLabel="tracking"
+                <ToTripButton ariaLabel="followyourouteonscreen" 
                   buttonClickAction={() => {
```
Comment (vesameskanen) [inline app/component/itinerary/ItineraryDetails.js:300]: This dedicated button does not need external arialabel prop.

File: app/translations.js [hunk 3]
```diff
@@ -2295,6 +2298,9 @@ const translations = {
     'fetch-new-route': 'Hae uusi reitti',
     'few-seats-available': 'Joitakin istumapaikkoja vapaana',
     finnish: 'Suomi',
+    follow: 'Matkan seuranta',
+    followtheitinerary: 'Matkalle',
+    followyourouteonscreen: 'Löydä perille ohjatusti!',
```
Comment (vesameskanen) [inline app/translations.js]: Use more consistent id:s. Route means P, IC455, bus 500 etc. For example:
- navigation-header
- start-navigation
- navigation-description

**Review by vesameskanen — CHANGES_REQUESTED** [review]

File: app/component/itinerary/ItineraryDetails.js (hunk 2 above)
Comment (SamiPuttonen) [reply app/component/itinerary/ItineraryDetails.js:300]: ariaLabel removed.

File: app/component/itinerary/ItineraryDetails.js (hunk 1 above)
Comment (SamiPuttonen) [reply app/component/itinerary/ItineraryDetails.js]: Button name changed.

File: app/translations.js (hunk 3 above)
Comment (SamiPuttonen) [reply app/translations.js]: Let's use those ones.

**Pull request merged**